database/src/lib.rs

```diff
@@ -594,6 +631,7 @@ pub enum DbLabel {
         scenario: Scenario,
         backend: CodegenBackend,
         metric: Metric,
```

Comment:
Could you please also keep the `target` before `metric` here, to keep `metric` at the end?